sklearn/impute/tests/test_impute.py

```diff
@@ -445,6 +445,17 @@ def test_imputation_constant_pandas(dtype):
     assert_array_equal(X_trans, X_true)
 
 
+@pytest.mark.parametrize('imputer', (SimpleImputer, IterativeImputer))
+def test_imputation_missing_value_in_test_array(imputer):
+    # [Non Regression Test] Missing value in test set should not throw an error
```

Comment:
```suggestion
    # [Non Regression Test for issue #13968] Missing value in test set should not throw an error
```

Reply:
@fhoang7 Could you accept this suggestion. This is handy to have the issue number for non-regression test

Reply:
Done!